sdccc/src/main/java/com/draeger/medical/sdccc/sdcri/testclient/TestClientUtil.java

```diff
@@ -70,7 +71,8 @@ public TestClientUtil(
             final LocalAddressResolver localAddressResolver,
             @Named(TestSuiteConfig.NETWORK_MULTICAST_TTL) final Long multicastTTL,
             @Named(TestSuiteConfig.TLS_ENABLED_PROTOCOLS) final String[] enabledTlsProtocols,
-            @Named(TestSuiteConfig.TLS_ENABLED_CIPHERS) final String[] enabledCiphers) {
+            @Named(TestSuiteConfig.TLS_ENABLED_CIPHERS) final String[] enabledCiphers,
+            @Named(TestSuiteConfig.CONFIGURATION_MODULE) final AbstractConfigurationModule configurationModule) {
```

Comment:
The TestSuite does not work anymore, because there is no default binding for this.

Reply:
done